roadmap/roadmap.adoc

```diff
@@ -0,0 +1,158 @@
+include::../shared/issue-footnotes.adoc[]
+
+:m3: ../metametamodel/metametamodel
+:refarch: ../reference-architecture/reference-architecture
+:serialization: ../serialization/serialization
+
+= LionWeb Roadmap, Releases, and History
+:toc:
+:toclevels: 4
+// :sectnums:
+// :sectnumlevels: 4
+
+== Roadmap
+As a volunteer-run Open Source effort, we cannot commit to a timeline.
+We're actively looking for corporate sponsors, with some prospects in sight.
+
+=== In progress
+
```

Comment:
Maybe something along the lines of finalizing the `2023.1` release, for completeness' sake?

Reply:
added